TraceListWidget.py

```diff
@@ -44,11 +44,11 @@ def popupMenu(self, pos):
         if (item == None): 
             dataaddAction = menu.addAction('Add Data Set')
 
-        action = menu.exec_(self.mapToGlobal(pos))
-        if action == dataaddAction:
-            dvlist = DataVaultList(self.parent.name)
-            self.windows.append(dvlist)
-            dvlist.show()
+            action = menu.exec_(self.mapToGlobal(pos))
+            if action == dataaddAction:
+                dvlist = DataVaultList(self.parent.name)
+                self.windows.append(dvlist)
+                dvlist.show()
 
```

Comment:
what is going on here? can you explain why this logic need to be inside the if (item==None): statement?

Reply:
If this logic is not inside of if (item == None), dataaddAction is not defined when python executes line 48. I think I didn't change any logic here. The original code uses a mixture of tab and spaces so the white space offset in the old code may not show the actual logic of the code.

Reply:
If this logic is not inside of if (item == None), dataaddAction is not defined when python executes line 48. I think I didn't change any logic here. The original code uses a mixture of tab and spaces so the white space offset in the old code may not show the actual logic of the code.